src/UniStaker.sol

```diff
@@ -218,6 +148,26 @@ contract UniStaker is INotifiableRewardReceiver, Multicall, EIP712, Nonces {
     emit RewardNotifierSet(_rewardNotifier, _isEnabled);
   }
 
+  /// @inheritdoc IUniStaker
+  function deposits(DepositIdentifier depositId) external view override returns (Deposit memory) {
+    return _deposits[depositId];
+  }
+
+  /// @inheritdoc IUniStaker
+  function surrogates(address delegatee) external view override returns (DelegationSurrogate) {
+    return _surrogates[delegatee];
+  }
```

Comment:
we can shrink the diff if we're okay with autogenerated getters returning the unpacked struct

confirmed with @gretzke that he's OK with the unpacked ABI